menus/sync-settings.cson

```diff
@@ -8,6 +8,7 @@
         { 'label': 'Backup', 'command': 'sync-settings:backup' }
         { 'label': 'Restore', 'command': 'sync-settings:restore' }
         { 'label': 'View backup', 'command': 'sync-settings:view-backup' }
+        { 'label': 'Check for updated backup', 'command': 'sync-settings:check-backup' }
```

Comment:
Not sure about the label, but can't think of a better one...
*  `Compare with backup`
* `Check changes`
* `Check for backup changes`